app/scripts/models/order.coffee

```diff
@@ -0,0 +1,84 @@
+root = exports ? this
+
+unt = require './unit'
+
+# TODO(cosmic): Locations need a class for normalization.
+root.Order = class Order
+  constructor: (order) ->
+    # Actor unit.
+    @unit = order.unit
+    # Action.
+    @act = order.act
+    # Source unit.
+    @src = order.src
+    # Destination.
+    @dst = order.dst
+    # String from which order was created (if any)
+    @str = order.str
+    # Support actions needs to be lent to a specific order.
+    # In particular, we can support holds.
+    @sup = order.sup
+    # Does order fail, and if so, why?
+    @fail = order.fail
+
+  @fromString: (str) ->
```

Comment:
Tests added. Due to how orders are formulated, the action word is not always in the beginning. The parser seems pretty reasonable to me. Are your objections to it going in as is strong?